azure-pipelines/generate-target-database.yaml

```diff
@@ -32,10 +32,7 @@ jobs:
 
       - script: |
           python -m pip install --upgrade pip wheel setuptools
-          pip install pipenv
-          python -m pipenv lock --dev-only -r --pre > dev-requirements.txt
-          pip install -r dev-requirements.txt
-          pip install pytest-azurepipelines
+          pip install -r requirements-test.txt
```

Comment:
Unlike the other commits, "ci: remove pipenv from generate-target-database.yaml" does indeed remove use of pipenv: this is good. Unlike the other commits, the first word after the change area is lowercase: we should use "ci: Remove pipenv from generate-target-database.yaml" to make this PR self-consistent.